specs/rollup-node.md

```diff
@@ -72,7 +117,11 @@ The input and return types here are as defined by the [engine API specs][engine-
   1. `blockNumber`: `QUANTITY`, 64 bits - L2 integer block number.
 - returns:
   1. `version`: `DATA`, 32 Bytes - the output root version number, beginning with 0.
-  1. `l2OutputRoot`: `DATA`, 32 Bytes - the output root.
+  1. `outputRoot`: `DATA`, 32 Bytes - the output root.
+  1. `blockRef`: `Object` - instance of [`L1BlockRef`](#l1blockref).
```

Comment:
This appears to be a L2BlockRef - it includes a `l1origin` field and `sequencerNumber`